pkg/enemy/damage.go

```diff
@@ -88,7 +88,7 @@ func (t *Enemy) calc(atk *combat.AttackEvent, evt glog.Event) (float64, bool) {
 	emBonus := (2.78 * em) / (1400 + em)
 	var reactBonus float64
 	//check melt/vape
-	if atk.Info.Amped {
+	if atk.Info.AmpType == combat.Melt || atk.Info.AmpType == combat.Vaporize {
```

Comment:
Should add another flag here. Don't use amp flag

Reply:
What should I called the flag? Catalyzed?